src/components/Quiz/QuizzesStats.tsx

```diff
@@ -0,0 +1,193 @@
+          gap={4}
+          bg="ednBackground"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={{ base: 8, lg: 12 }}
+          mb={-2}
+        >
+          <GridItem colSpan={{ base: 2, lg: 1 }} alignSelf="center" order={1}>
+            <Text
+              color="body"
+              fontWeight="bold"
+              fontSize="xl"
+              margin={0}
+              textAlign={{ base: "center", lg: "left" }}
+            >
+              <Translation id="your-total" />
+            </Text>
+          </GridItem>
+
+          <GridItem
+            colSpan={{ base: 2, lg: 1 }}
+            justifySelf={{ base: "auto", lg: "end" }}
+            alignSelf="center"
+            order={{ base: 3, lg: 2 }}
+          >
+            <Button
+              variant="outline-color"
+              leftIcon={<Icon as={FaTwitter} />}
+              onClick={() =>
+                handleShare({ score: userScore, total: TOTAL_QUIZZES_POINTS })
+              }
+              w={{ base: "full", lg: "auto" }}
+              mt={{ base: 2, lg: 0 }}
+            >
+              <Translation id="share-results" />
+            </Button>
+          </GridItem>
+
+          <GridItem colSpan={2} order={{ base: 2, lg: 3 }}>
+            <Stack gap={2}>
+              <Flex
+                justifyContent={{ base: "center", lg: "flex-start" }}
+                alignItems="center"
+              >
+                <Circle size="64px" bg="primary" mr={4}>
+                  <TrophyIcon color="neutral" w="35.62px" h="35.62px" />
+                </Circle>
+
+                <Text fontWeight="bold" fontSize="5xl" mb={0} color="body">
+                  {userScore}
+                  <Text as="span" color="bodyLight">
+                    /{TOTAL_QUIZZES_POINTS}
+                  </Text>
+                </Text>
+              </Flex>
+
+              <Progress value={userScore} />
+
+              <Flex direction={{ base: "column", lg: "row" }}>
+                <Text mr={10} mb={0} mt={{ base: 2, lg: 0 }} color="bodyLight">
+                  <Translation id="average-score" />{" "}
+                  <Text as="span" color="body">
+                    {average}%
+                  </Text>
+                </Text>
+
+                <Text mb={0} color="bodyLight">
+                  <Translation id="completed" />{" "}
+                  <Text as="span" color="body">
+                    {numberOfCompletedQuizzes}/{totalQuizzesNumber}
+                  </Text>
+                </Text>
+              </Flex>
+            </Stack>
+          </GridItem>
+        </Grid>
+
+        {/* community stats */}
+        <Flex
+          direction="column"
+          gap={6}
+          justifyContent="space-between"
+          bg="ednBackground"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={{ base: 8, lg: 12 }}
+        >
+          <Text fontWeight="bold" fontSize="xl" mb={0}>
```

Comment:
```suggestion
          <Text color="body" fontWeight="bold" fontSize="xl" mb={0}>
```

<img width="419" alt="Screenshot 2023-05-29 at 09 25 39" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/0aa41037-ab5d-4627-9490-bb2004de6cfc">